test/test_gipc.py

```diff
@@ -269,13 +269,19 @@ def test_pid(self):
         _call_close_method_if_exists(p)
 
     def test_terminate(self):
-        # 1s is too long, and the process is killed on python 3.8.6
-        p = start_process(gevent.sleep, args=(0.5,))
+        p = start_process(gevent.sleep, args=(1.5,))
```

Comment:
Note: this relates to a conversation between myself and @tyler-willey from a while ago: https://github.com/jgehrcke/gipc/pull/110/files#r644652376